ipa-core/src/query/executor.rs

```diff
@@ -101,6 +101,22 @@ pub fn execute<R: PrivateKeyRegistry>(
         }
         // TODO(953): This is really using BA32, not Fp32bitPrime. The `FieldType` mechanism needs
         // to be reworked.
+        #[cfg(any(test, feature = "cli", feature = "test-fixture"))]
+        (QueryType::OprfIpaRelaxedDpPadding(ipa_config), FieldType::Fp32BitPrime) => do_query(
+            config,
+            gateway,
+            input,
+            move |prss, gateway, config, input| {
+                let ctx = SemiHonestContext::new(prss, gateway);
+                Box::pin(
+                    OprfIpaQuery::<BA32, R>::new(ipa_config, key_registry)
+                        .execute(ctx, config.size, input)
+                        .then(|res| ready(res.map(|out| Box::new(out) as Box<dyn Result>))),
+                )
+            },
+        ),
```

Comment:
I'm confused by this warning. it seems like the test coverage should be (intentionally) missing for lines 120-131. but otherwise this is fine, we are doing this specifically to not add full padding in tests.

Reply:
it seems like we test this elsewhere? i'm not immediately familiar with exactly how this is tested (or if we'd expect codecov to catch it)

Reply:
I think it was shown as missing before for other QueryTypes so copying one of those to make a new one just increased the ratio past 20%.  So I don't think we're really missing some tests that other QueryTypes have somewhere.